docs/general/builtins/aggregation.rst

```diff
@@ -832,31 +832,103 @@ Example::
+    | topk(country, 3)                                                                                                                                                                                                                                                 |
+    +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+    | {"frequencies": [{"estimate": 436, "item": "IT", "lower_bound": 436, "upper_bound": 436}, {"estimate": 401, "item": "AT", "lower_bound": 401, "upper_bound": 401}, {"estimate": 320, "item": "CH", "lower_bound": 320, "upper_bound": 320}], "maximum_error": 0} |
+    +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Internally a `Frequency Sketch`_ is used to track the frequencies of the most
+common values. Higher values in max_capacity provide better accuracy at the
+cost of increased memory usage. If more items than 75 % of the capacity are
+processed the frequencies of the result are an approximation. The result
+contains then all values with their frequencies above the error threshold and
+may also include false positives. The error threshold indicates the minimum
+size of frequency which can be detected reliably.The following table is an
+extract of the `Error Threshold Table`_ and shows the error threshold in
+relation to the capacity and the number of distinct tracked items.
+
+.. list-table:: Frequency Threshold
+   :widths: 20 20 20 20 20 20 20 20
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Max Capacity / Number of distinct items
+     - 8192
+     - 16384
+     - 32768
+     - 65536
+     - 131072
+     - 262144
+     - 524288
+   * - 10000
+     - 4
+     - 0
+     - 0
+     - 0
+     - 0
+     - 0
+     - 0
+   * - 100000
+     - 43
+     - 21
+     - 11
+     - 5
+     - 3
+     - 0
+     - 0
+   * - 1000000
+     - 427
+     - 214
+     - 107
+     - 53
+     - 27
+     - 13
+     - 7
+   * - 10000000
+     - 4272
+     - 2136
+     - 1068
+     - 534
+     - 267
+     - 134
+     - 67
+   * - 100000000
+     - 42725
+     - 21362
+     - 10681
+     - 5341
+     - 2670
+     - 1335
+     - 668
+   * - 1000000000
+     - 427246
+     - 213623
+     - 106812
+     - 53406
+     - 26703
+     - 13351
+     - 6676
+
+
+The error threshold shows which ranges of frequencies can be tracked depending
+on the number of distinct items and max capacity. E.g. with the default max_capacity of
+8192 frequencies higher than 4 can be tracked on 10000 distinct items reliable.
+A threshold of 0 indicates that the frequencies are exact.
```

Comment:
looking at https://datasketches.apache.org/docs/Frequency/FrequentItemsErrorTable.html
This is not talking about distinct elements in the stream, but rather the total amount of elements processed in the stream

In the scenario where you have processed 10,000 items with a max capacity of 8192:
- 	The threshold  T  is about 4.27.
- 	All items with frequencies greater than 4 will be included.
- 	Some items with frequencies below the threshold 4 may also appear in the result.

Reply:
You are right, thanks for the feedback